DQMOffline/Trigger/plugins/TopMonitor.cc

```diff
@@ -941,6 +980,11 @@ void TopMonitor::fillDescriptions(edm::ConfigurationDescriptions & descriptions)
   histoPSet.add<edm::ParameterSetDescription>("lsPSet", lsPSet);
 
   desc.add<edm::ParameterSetDescription>("histoPSet",histoPSet);
+  //Suvankar
+  edm::ParameterSetDescription lPVcutPSet;
+  lPVcutPSet.add<double>( "dxy", 0. );
+  lPVcutPSet.add<double>( "dz",  0. );
```

Comment:
@sroychow we would prefer to have these two parameters set to some safe value, i.e. some value that does not cut (for example 99999 or something like that). I know it probably sounds redundant given you created a dedicated flag, but it would make the code more safe and understandable